packages/office-ui-fabric-react/src/components/Keytip/examples/Keytips.Dynamic.Example.tsx

```diff
@@ -1,64 +1,39 @@
 import * as React from 'react';
 import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
 
-interface IKeytipsTestState {
-  currButton: string;
-}
+const keytipProps = {
```

Comment:
I'm not sure this is correct if you want to preserve the previous behavior of having buttons 1 and 2 have different keytips.